src/components/Home.tsx

```diff
@@ -18,8 +16,8 @@ export default function Home() {
       setShow(true);
       setTimeout(() => {
         setBackgroundImage("blue");
-      }, 1000);
-    }, 1000);
+      });
+    });
```

Comment:
its removed because this causes your UI to break. It basically wait 1 sec to display anything. But to use the TimeOuts is also not a good idea, we'll fix that later